__tests__/reducers/app.js

```diff
@@ -8,6 +8,7 @@ describe('Reducer: App', () => {
       AST: {},
       filters: {},
       hasUnsavedChanges: false,
+      language: 'python',
```

Comment:
We might need to save parser lang and Codemirror lang seperately, as the parser we need to pull is `python3`